aries_cloudagent/messaging/jsonld/tests/test_credential.py

```diff
@@ -1,15 +1,38 @@
-import json
+"""Test json-ld credential."""
 
+import json
 from asynctest import mock as async_mock, TestCase as AsyncTestCase
-
+import pytest
+import asyncio
 from .. import credential as test_module
+from itertools import cycle
+from ....admin.request_context import AdminRequestContext
+from ..credential import (
+    verify_credential,
+    sign_credential,
+    did_key,
+)
+from ..error import SignatureTypeError, BadJWSHeaderError
+from ..create_verify_data import DroppedAttributeError
+from ....core.in_memory import InMemoryProfile
+from ....wallet.in_memory import InMemoryWallet
+from ....wallet.base import BaseWallet
+from . import (
+    TEST_SEED,
+    TEST_VERKEY,
+    TEST_VERIFY_OBJS,
+    TEST_SIGN_OBJS,
+    TEST_SIGN_ERROR_OBJS,
+    TEST_VALIDATE_ERROR_OBJ2,
+    TEST_VERIFY_ERROR,
+)
 
-TEST_VERKEY = "5yKdnU7ToTjAoRNDzfuzVTfWBH38qyhE1b9xh4v8JaWF"
+TEST_VERKEY_ = "5yKdnU7ToTjAoRNDzfuzVTfWBH38qyhE1b9xh4v8JaWF"
```

Comment:
This is an odd change; can we remove this and use the imported `TEST_VERKEY`?

Reply:
I didn't want to change the tests from before, but I did want to use the TEST_VERKEY variable, so I added appended a _. should I update the older tests to use the new test data?